app/assets/javascripts/d3/merit_transformator.js

```diff
@@ -14,25 +14,17 @@ var MeritTransformator = (function () {
     MeritTransformator.prototype = {
         transform: function () {
             this.data.forEach(function(serie) {
-                if (!(serie.oldValues)) {
-                    serie.oldValues = serie.values;
-                }
-
                 serie.values = transformSerieValues.call(this, serie);
             }.bind(this));
 
 
-            if (this.scope.stack) {
-                return this.scope.stack(this.data);
-            } else {
-                return this.data;
-            }
+            return this.data;
         }
     };
 
     function MeritTransformator (scope, data) {
         this.scope = scope;
-        this.data  = data;
+        this.data  = $.extend(true, [], data);
```

Comment:
@antw this is now the slowest line by far now. Only thing is that we could maybe do a custom `$.extent` to maybe shave of 100ms or so? I'm a bit wondering if that'll help a lot though..